section4/exercises/classMethods.js

```diff
@@ -28,13 +28,27 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
-  // ADD CODE
+changeProtein() {
```

Comment:
I think the instructions mention an argument should be passed into this `changeProtein` method, similar to the `gainPower` method from your SuperHero class.